src/helpers/animations.js

```diff
@@ -0,0 +1,20 @@
+// Function wrapping code.
+// fn - reference to function
+// context - what you want "this" to be.
+// params - array of parameters to pass to function.
+var wrapFunction = function (fn, context, params) {
+  return function () {
+    fn.apply(context, params);
+  };
+};
+
+// Create a function to be wrapped
+export const moveBall = function (ballRef, start, end) {
```

Comment:
Note: could try to pass the `clock.getElapsedTime()` as a function argument, since it's not possible to store it in a global variable...